test/e2e/quick_start_test.go

```diff
@@ -307,6 +309,34 @@ var _ = Describe("Quick start", func() {
 														),
 													},
 												)
+
+												if os.Getenv("RUN_CIS_BENCHMARK") == "true" {
+													By("Running CIS benchmark against workload cluster")
+
+													trivyCmd := exec.Command( //nolint:gosec // Only used for testing so safe here.
+														"trivy",
+														"k8s",
+														"--compliance=k8s-cis-1.23",
+														"--disable-node-collector",
+														"--report=summary",
+														fmt.Sprintf(
+															"--output=%s",
+															filepath.Join(
+																os.Getenv("GIT_REPO_ROOT"),
+																"cis-benchmark-report.txt",
+															),
+														),
+														fmt.Sprintf(
+															"--kubeconfig=%s",
+															workloadProxy.GetKubeconfigPath(),
+														),
+													)
+
+													trivyCmd.Stdout = GinkgoWriter
+													trivyCmd.Stderr = GinkgoWriter
+
+													Expect(trivyCmd.Run()).To(Succeed(), "CIS benchmark failed")
```

Comment:
How are tests passing here if some CIS benchmarks are not yet fixed? 🤔

Reply:
Because I can't get it to fail! It's reporting only...

Reply:
Got it.